content/journals.md

```diff
@@ -64,33 +54,17 @@ title: Journals
     [Homepage](https://www.springer.com/journal/11430)
 - **Solid Earth**:
     [Homepage](https://www.solid-earth.net/index.html)
-- **Tectonics**:
-    [Homepage](https://agupubs.onlinelibrary.wiley.com/journal/19449194)
```

Comment:
We may want to keep this since it is a good AGU journal and some seismologists studying seismic imaging may use it.